types/es-to-primitive/.editorconfig

```diff
@@ -0,0 +1,3 @@
+# This package uses tabs
```

Comment:
Could you please remove this file?

Reply:
If I do this, then it’s very likely that someone would change tabs to spaces under me.

Also, all my DT contributions have this file.